src/helper/service-utils.php

```diff
@@ -279,6 +279,15 @@ function generate_global_docker_compose_yml( Filesystem $fs ) {
 				'global-backend-network',
 			],
 		],
+		[
+			'name'    => EE_CRON_SERVICE,
+			'image'   => 'easyengine/cron:' . $img_versions['easyengine/cron'],
+			'restart' => 'always',
+			'volumes' => [
+				'/opt/easyengine/services/cron:/etc/ofelia:ro',
```

Comment:
Use function: `EE_DOCKER::get_mounting_volume_array()` as in all of above to generate volumes. As the volume generation differs in Linux and macOS. 

For the volume `/var/run/docker.sock` -  You can use `skip_volume`:  https://github.com/EasyEngine/service-command/blob/00135719110c515bf01121883b070e2d4027b98d/src/helper/service-utils.php#L155-L160

Once, this change is done, for Linux systems, the cron directory inside the EE_SERVICE_DIR will be a docker volume. So, you'll have to write migrations to take backup of the previous cron directory and then once volume directory get's symlinked, need to restore that data.